ledger/participant-integration-api/src/main/scala/platform/store/dao/events/TransactionLogUpdatesConversions.scala

```diff
@@ -10,66 +10,82 @@ import com.daml.ledger.api.v1.transaction.{
   TreeEvent,
   Transaction => FlatTransaction,
 }
+import com.daml.ledger.api.v1.transaction_service.{
+  GetTransactionTreesResponse,
+  GetTransactionsResponse,
+}
 import com.daml.ledger.api.v1.{event => apiEvent}
 import com.daml.lf.data.Ref
+import com.daml.lf.data.Ref.{Identifier, Party}
+import com.daml.lf.value.Value.ContractId
 import com.daml.logging.LoggingContext
-import com.daml.platform.{ApiOffset, ContractId, FilterRelation, Identifier, Value}
 import com.daml.platform.api.v1.event.EventOps.TreeEventOps
 import com.daml.platform.participant.util.LfEngineToApi
-import com.daml.platform.store.interfaces.TransactionLogUpdate
 import com.daml.platform.store.interfaces.TransactionLogUpdate.{CreatedEvent, ExercisedEvent}
+import com.daml.platform.store.interfaces.{TransactionLogUpdate => TxUpdate}
+import com.daml.platform.{ApiOffset, FilterRelation, Value}
 import com.google.protobuf.timestamp.Timestamp
 
 import scala.concurrent.{ExecutionContext, Future}
 
 private[events] object TransactionLogUpdatesConversions {
+  type Filter = TxUpdate.Transaction => Option[TxUpdate.Transaction]
```

Comment:
Hmm, it's less verbose when referenced in other places. But fine by me to de-alias.

Reply:
Not sure what you are referring to. On any rate my comment is general in that sense: if it is not possible to extract it is fine, if it would be possible, next time I would consider to extract that. Quality of review can be improved if meaningful changes do not collide with pure refactorings.

Reply:
I understand the need for more complex types, I am merely raising this is not so complex. The tradeoff is the mental mapping what the author forces on the reviewer.